new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/WebLogicImageTool.java

```diff
@@ -55,30 +65,45 @@ public boolean updateImage() {
     } 
 
     try {
-      checkFile(IMAGE_TOOL);
+      // delete the old cache entry for the WDT installer
+      if (!deleteEntry()) {
+        logger.warning("Failed to delete cache entry in WebLogic Image Tool");
```

Comment:
Why warnings?  These should either be asserts that fail the test or they aren't important.

Reply:
This is part of test infra, not a test. The pattern we have been following is for the infra part to return success or not. The test code, such as ItWITValication, will then fail the test with assertions. But my understanding might be wrong here.